src/commands/job/execute.ts

```diff
@@ -68,7 +89,7 @@ export async function execute(
     )
   }
 
-  const contextName = getContextName(target, returnStatusOnly)
+  const contextName = await getContextName(target, returnStatusOnly)
```

Comment:
@allanbowe / @YuryShkoda I don't know how this was _ever_ working before.

Reply:
clearly we need better tests!